tests/unit/CoreHeadingTest.php

```diff
@@ -226,21 +226,21 @@ public function test_retrieve_core_heading_with_font_and_anchor() {
 		$this->assertEquals(
 			[
 				'align'           => null,
-				'anchor'          => 'custom-id',
+				'anchor'          => 'custom-id', // Previously untested
 				'backgroundColor' => null,
 				'className'       => null,
 				'content'         => 'Custom Font Heading',
 				'cssClassName'    => 'wp-block-heading',
-				'fontFamily'      => null, // Previously untested
-				'fontSize'        => null,
+				'fontFamily' => null, // @todo: This is not returning correctly.
+				'fontSize' => null, // @todo: This is not returning correctly.
```

Comment:
Please copy `phpcs.xml.dist` to `phpcs.xml`, comment out the exclude-pattern for tests, and _remember to format_ before you commit ;-)